### DISCUSSION TITLE
Getting the current stock quantity for a specific product

### DISCUSSION THREAD

**Discussion by ayoubAnbara** [amzn/selling-partner-api-models#3738] [General]
I use getListingItem Api to get product information, here is the examples of the HTTP responses:
```
{
    "sku": "xyz",
    "fulfillmentAvailability": [
        {
            "fulfillmentChannelCode": "AMAZON_EU"
        }
    ]
}
```
```
{
    "sku": "abcd",
    "fulfillmentAvailability": [
        {
            "fulfillmentChannelCode": "DEFAULT",
            "quantity": 0
        }
    ]
}
```
the field **fulfillmentChannelCode** can have different values, I want what is the meaning of these values, to know the stock quantity of a product fulfilled by the seller.
Ref: [doc](https://developer-docs.amazon.com/sp-api/docs/listings-items-api-v2021-08-01-reference#fulfillmentavailability)

**Comment by stefnats** [comment]
I hope i get your question right:

`fulfillmentChannelCode` stands for
- `DEFAULT` = Merchant fulfilled
- `AMAZON_EU` / `AMAZON_NA` / whatever starts with `AMAZON_` is fullfilled by Amazon (FBA / AFN)

If you need the stock / quantities for FBA Listings, please check out the [FBA Inventory API](https://developer-docs.amazon.com/sp-api/docs/fbainventory-api-v1-reference)

**Reply by ayoubAnbara** [reply]
Thank you @stefnats  , I will test that, and get back to you.
